cart/render-line-items.js

```diff
@@ -0,0 +1,32 @@
+import { plants as plantData } from '../data.js';
+import { findById } from '../utils.js';
+
+
+export function renderTableRow(cartItem) {
+
+    const tr = document.createElement('tr');
+    const tdItem = document.createElement('td');
+    const tdPrice = document.createElement('td');
+    const tdQuantity = document.createElement('td');
+    const tdSubtotal = document.createElement('td');
+
+    tdQuantity.textContent = cartItem.quantity;
+
+    const plantObject = findById(plantData, cartItem.id);
```

Comment:
Kuddos on the variable names, I've said this before but the better names you have the easier it is to read and understand. Continue to do this.